osu.Game/Overlays/TabControlOverlayHeader.cs

```diff
@@ -30,6 +31,16 @@ public Bindable<T> Current
             set => current.Current = value;
         }
 
+        protected new float ContentSidePadding
```

Comment:
this is perhaps a little strange but we have so many conventions for these types of things (get-only expression-bodied member/bindable/property with setter) at this point that I guess it can stay.